configuration-sample/update

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+set -eu
+
+# Get directory of this script
+scriptDir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# Expected location of ods-configuration
+cd "$scriptDir/../.."
+odsConfigurationLocation="$(pwd)/ods-configuration"
+cd - > /dev/null
+
+echo "Checking if ods-configuration repository exists next to ods-core ..."
+if [ ! -d "$odsConfigurationLocation" ]; then
+  echo "> Directory $odsConfigurationLocation does not exist yet. It will be created now."
+  mkdir -p $odsConfigurationLocation
+  echo "> Created directory $odsConfigurationLocation."
+else
+  echo "> Directory $odsConfigurationLocation exists."
+fi
+
+echo ""
+
+echo "Updating .env.sample files ..."
+cp *.env.sample $odsConfigurationLocation/
+echo ">.env.sample files updated."
+
+echo ""
+
+echo "Checking for missing params in .env files ... "
+cd $odsConfigurationLocation
+elementIn() {
+  local e match="$1"
+  shift
+  for e; do [[ "$e" == "$match" ]] && return 0; done
+  return 1
+}
+anyDrift=false
+for i in *.env.sample; do
+  actualFile="${i%%.sample}"
+  if [ ! -f "$actualFile" ]; then
+    echo "> WARNING! Actual param file $actualFile does not exist yet and will be created. Please review its contents carefully."
+    cp $i $actualFile
+  else
+    sampleParams=$(cat $i | grep "[A-Z1-9_]\+=" | awk -F'=' '{print $1}')
+    actualParams=($(cat $actualFile | grep "[A-Z1-9_]\+=" | awk -F'=' '{print $1"="}'))
+    for sampleParam in $sampleParams; do
+      if ! (elementIn "${sampleParam}=" "${actualParams[@]}"); then
+        anyDrift=true
+        echo "> WARNING! $sampleParam is present in $i, but not in $actualFile. Please add it."
+      fi
+    done
+  fi
```

Comment:
Should the script also highlight keys no longer needed?

Reply:
@henrjk It could, but I did not consider that as very critical so would rather have that as a follow-up. Feel free to open an issue for it!